Server/MirObjects/NPC/NPCSegment.cs

```diff
@@ -236,11 +236,6 @@ public void ParseCheck(string line)
                     CheckList.Add(new NPCChecks(CheckType.CheckMap, parts[1]));
                     break;
 
-                case "MAPLIGHT":
```

Comment:
Looks like this PR hasn't brought through the previous map light additions. This needs pulling in otherwise it'll merge it out.